controllers/activemqartemis_controller_deploy_operator_test.go

```diff
@@ -169,6 +171,59 @@ var _ = Describe("artemis controller", Label("do"), func() {
 		})
 	})
 
+	Context("operator deployment in default namespace", Label("do-operator-restricted"), func() {
```

Comment:
I don't think that label is accurate. maybe delete or change. or change both to deploy-operator?

Reply:
I changed that label to `do-operator-with-custom-related-images`